CMakeLists.txt

```diff
@@ -4,6 +4,19 @@ project(Darkflame
 	LANGUAGES C CXX
 )
 
+# TEMP - Sanitizer flags
+if (UNIX AND NOT APPLE)
+	# add_compile_options("-fsanitize=address,undefined" "-fvisibility=default")
+	add_compile_options("-fsanitize=undefined" "-fvisibility=default")
+	add_link_options("-fsanitize=undefined")
+
+	if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+		add_link_options("-static-libsan")
+	else()
+		add_link_options("-static-libasan")
+	endif()
+endif()
```

Comment:
we should keep these and lock them under a cmake variable flag

Reply:
Agreed. the 'TEMP' bit is just a reminder to myself that I need to create a CMake config to enable/disable these